src/components/structures/FontSlider.js

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2020 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Slider extends React.Component {
```

Comment:
This should also have a `@replaceableComponent("structures.Slider")` definition because it's not TypeScript

Reply:
Yeah sure. I've moved this one over to it. Is there a plan for reducing the redundency of having two type declerations through both react's prop-types and typescript's anotiations?

Reply:
We're using TypeScripts annotations (`extends React.Component<IProps, IState>`) instead of `propTypes`